ceph-dev-pipeline/build/Jenkinsfile

```diff
@@ -443,14 +442,15 @@ pipeline {
                 def deb_build_profiles = "";
                 switch (env.FLAVOR) {
                   case "default":
-                    ceph_extra_cmake_args += " -DALLOCATOR=tcmalloc"
+                    ceph_extra_cmake_args += " -DALLOCATOR=tcmalloc -DWITH_CRIMSON=true"
```

Comment:
maybe it's time for [CMakeLists.txt](https://github.com/ceph/ceph/blob/cf11bef497/CMakeLists.txt#L740) to enable this option by default?

[ceph.spec.in](https://github.com/ceph/ceph/blob/cf11bef497/ceph.spec.in#L94) could do similar with
```diff
-%bcond_with crimson
+%bcond_without crimson
```
that way ceph-build doesn't need to do anything special for crimson

Reply:
Makes sense, cmake could also verify that the compiler used is capable of building crimson and the distro used. I'll follow up on this once the dependency packaging PR is merged. Thanks!